rubicon/CHANGES.md

```diff
@@ -0,0 +1,20 @@
+# 2.1.4
+ 
+- Testing file:
+the test file has been added with 8 tests and 8 success (system-tester.html).
+npm run debug  and reach /system-tester.html
+result: 8 specs, 0 failures
+ 
+- ESLint:
```

Comment:
Remove this line from the `.eslintignore` file.
https://github.com/AntoineJac/ix-library-adapters/blob/canary/.eslintignore#L15

Reply:
Done. Thanks,